src/styles/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  padding: 50px 40px;
+  margin: 0 auto;
+  max-width: 1024px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) 11th item. Let's use some type of calculationusing card-width, gap between cards and max number of cards in a row. So you need to create a handful of vars and use them